daprdocs/content/en/reference/cli/dapr-init.md

```diff
@@ -54,6 +57,27 @@ Dapr can also run [Slim self-hosted mode]({{< ref self-hosted-no-docker.md >}})
 dapr init -s
 ```
 
+You can also specify a private registry to pull container images from. These images needs to be published to private registries in below fashion to let dapr init pull them successfully - 
+
+1. Placement container image(dapr) - dapr/dapr:<version>
+2. Redis container image(rejson)   - dapr/3rdparty/rejson
+3. zipkin container image(zipkin)  - dapr/3rdparty/zipkin
+
+> As you can see above, all required images by Dapr needs to be under `dapr` directory.
+
+> The 3rd party images have to be published under `dapr/3rdparty` directory;
+
+> image-registy uri follows this format - `example.io/<owner|username>`
+
+```bash
+dapr init --image-registry example.io/<owner|username>
```

Comment:
Is is possible to use this URL syntax for --image-registry to pull from DockerHub rather than having to using the env variable? People may prefer this more explicit approach rather than having to set the env var? If possible, I suggest showing this as an example also.

Reply:
yes. It can be used like this  - `docker.io/username` . I am updating the docs with this.